lib/nrf_modem_lib/lte_net_if/lte_net_if.c

```diff
@@ -406,23 +399,8 @@ static void lte_net_if_init(struct conn_mgr_conn_binding *if_conn)
 
 int lte_net_if_enable(void)
 {
-	int ret;
-
 	if (!nrf_modem_is_initialized()) {
-		ret = modem_init();
-		if (ret) {
-			LOG_ERR("modem_init, error: %d", ret);
-			return ret;
-		}
-	}
-
-	/* Calling this function if the link controller has already been initialized is safe,
-	 * as it will return 0 in that case.
-	 */
-	ret = lte_lc_init();
-	if (ret) {
-		LOG_ERR("lte_lc_init, error: %d", ret);
-		return ret;
+		return nrf_modem_lib_init();
```

Comment:
Not sure, this just logging a call to `nrf_modem_lib_init`. Shall I add a log line there instead?

Reply:
I was thinking it would be useful to see a log and it would also indicate that `lte_net_if_enable()` had been called but seems `lte_net_if_disable()` is not logged either. Doesn't really matter for me...

Reply:
Let's remove it here then, I can add it do `nrf_modem_lib_init` to make sure we log these operations in `nrf_modem_lib` at least :)